### ISSUE TITLE
Keybindings configuration

### ISSUE THREAD

**Issue by pechorin** [pechorin/any-jump.nvim#6]
For now i'm a little confusing about keybindings config implementation

Where are several ways:

— user should explicitly set all keybindings in config by self
—user should add let g:any_jump_skip_default_keybindings and explicitly add keybinds, otherwise default will be set
—User can explicitly call call g:AnyJumpSetupKeybinds() (maybe with provided key configs for overriding defaults), otherwise no keybindings will set

The big area to think.

**Comment by wadkar** [comment]
Typically, vim plugins do _not_ add keybindings as it is considered rude to pollute/override keybindings.

I like the another approach where plugin defines keybindings but provides an option to enable it like `let g:any_jump_enable_keybindings` with default set to `0` (i.e. disabled by default).

In this approach a TL;DR/quick setup section at the top of README can get a user quickly up and running. For example,

---
### TL;DR

Add `Plug 'pechorin/any-jump.vim'` to your `vimrc`/`init.vim`, then open a file and place your cursor on a variable. Try `:AnyJumo` to jump to the variables definition. `:AnyJumpBack` to go back to previously opened file. `:AnyJumpLastResults` will open the last closed search window.

---

Cool plugin BTW!

👍 😎

**Comment by pechorin** [comment]
@wadkar thanks for your notes and explanation. As you notice currently i implement agressive keybinds setup :) Hope this is not very cruel for users.

**Comment by wadkar** [comment]
It's fine for me, as I tend to read the docs and configure any plugin before `:source`ing my `$MYVIMRC`. I am looking forward to the fzf integration that's being discussed in the feedback thread. Floating windows are awesome, and doubly so with fzf!

**Comment by pechorin** [comment]
if i will implement fzf support then this means what it will be separate ui :) maybe it's okay, will think about it. thanks for response :)

**Comment by blayz3r** [comment]
_I think this is the norm:_
— user can explicitly call call g:AnyJumpSetupKeybinds() (maybe with provided key configs for overriding defaults), otherwise no keybindings will set
